src/Raven.Server/Integrations/PostgreSQL/Handlers/PostgreSqlUsernames.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System.Collections.Generic;
+
+namespace Raven.Server.Integrations.PostgreSQL.Handlers
+{
+    public class PostgreSqlUsernames
+    {
+        public List<User> Users { get; set; }
+
+        public PostgreSqlUsernames()
+        {
+            Users = new List<User>();
+        }
+    }
+
+    public class User
```

Comment:
This is a *very* generic name. Can we rename that to something that won't likely collide with other things?

Reply:
Renamed to `PostgreSqlUsername` and pushed to already open PR:

https://github.com/ravendb/ravendb/pull/12989/commits/07e01aae221622dc916581adc3e5029eb0c03713